packages/mobile/src/identity/reducer.ts

```diff
@@ -46,8 +46,13 @@ export interface AddressToDataEncryptionKeyType {
   [address: string]: string | null // null means no DEK registered
 }
 
+export interface AddressInfoToDisplay {
+  name: string
+  picture: string | null
```

Comment:
What do you think to rename this to `imageUrl`?
To make it more explicit what this field is.